x/genutil/v2/cli/export.go

```diff
@@ -56,6 +57,10 @@ func ExportCmd(app ExportableApp) *cobra.Command {
 			}
 
 			height, _ := cmd.Flags().GetInt64(flagHeight)
+			forZeroHeight, _ := cmd.Flags().GetBool(flagForZeroHeight)
```

Comment:
do we have a test for both non zero and zero?

Reply:
We do, that it why there were failing.